echoshader/new_version/echogram.py

```diff
@@ -57,8 +72,28 @@ def _init_widget(self):
             step=0.01,
         )
 
-        self.color_map = panel.widgets.TextInput(
-            name="Color Map", value="jet", placeholder="jet"
+        self.tile_select = panel.widgets.Select(
+            name="Map Tile Select", value="OSM", options=get_tile_options()
+        )
+
+        self.curtain_ratio = panel.widgets.FloatInput(
+            name="Ratio Input (Z Spacing)", value=0.001, step=1e-5, start=0
+        )
+
+        self.link_mode_select = panel.widgets.Select(
+            name="Link Mode Select",
+            options={
+                "Track controled by Echogram": True,
+                "Echogram controled by Track": False,
+            },
+        )
```

Comment:
We discussed yesterday that it would be much more intuitive to use the echogram <--> map/curtain functionality _without_ this menu selector (so that user only needs to worry about the `update_echogram_button` and `update_positions_button`). Could you make a new issue to tackle after this PR is merged?

Reply:
Hey @leewujung, issue with reference: https://github.com/OSOceanAcoustics/echoshader/issues/126